### ISSUE TITLE
[Accessibility]: Repo Management - Can't connect code hsot from code host connections page 

### ISSUE THREAD

**Issue by jplahn** [sourcegraph/sourcegraph#36420]
### Audit type

Keyboard navigation

### User journey audit issue

https://github.com/sourcegraph/sourcegraph/issues/33520

### Problem description

There's no way to tab from the landing page on "code host connections" to the "Connect" button for either GitHub or Gitlab 

### Expected behavior

I should be able to use the tab button to Connect GitHub or gitlab from the "Code host connections" settings page 

### Assigning labels

- [ ] Please give this issue an estimate by applying a label like `estimate/Xd`, where X is the estimated number of days it will take to complete.
- [ ] If this issue is specific to a specific Sourcegraph product, please assign the appropriate team label to this issue.
- [ ] If this issue will require input from designers in order to complete, please assign the label `needs-design`.
- [ ] If you are confident that this issue should be fixed by GitStart, please assign the label `gitstart`.

### Owner

This issue will be fixed by my team, I have assigned a relevant member to it, or I will do so in the near future.

**Comment by sourcegraph-bot-2** [comment]
Heads up @jplahn - the "team/repo-management" label was applied to this issue.

**Comment by umpox** [comment]
As we have recently decided to sunset user-added repositories on [Sourcegraph.com](http://sourcegraph.com/), this audit flow is no longer available. 

Closing this issue as not planned.
